lib/ModalPresenter.tsx

```diff
@@ -85,12 +86,10 @@ const ModalPresenter = forwardRef<Ref, ViewProps>(
   },
 );
 
-const ModalPresenterWrapper: React.FC<ViewProps> = ({ children }) => {
+export const ModalPresenterParent: FC<{}> = ({ children }) => {
```

Comment:
shouldn't this be `export default ModalPresenterParent`

Reply:
this might be an open question. In my own opinion, when a module has a default export, I'd expect that to be the most commonly used component among all exports. If the module is mainly a view component, the default export can be that view. If the module is mainly a set of APIs, there shouldn't be a default export. In this case, although this parent component is a view, but it's not the focus of the whole usage of this library.

Reply:
I agree! merging it

Reply:
an update to "If the module is mainly a set of APIs, there shouldn't be a default export.", in this case, the default export can be an object which includes all the APIs.